cmd/credentials_test.go

```diff
@@ -15,6 +15,7 @@ func TestValidSelectorParsing(t *testing.T) {
 		"file://../tst/selectors/valid-all-attributes-selector.json",
 		"file://../tst/selectors/valid-some-attributes-selector.json",
 		"Key=x509Subject,Value=CN=Subject Key=x509Issuer,Value=CN=Issuer Key=x509Serial,Value=15D19632234BF759A32802C0DA88F9E8AFC8702D",
+		"Key=x509Subject,Value=CN=CN With Spaces Key=x509Issuer,Value=CN=CN With Spaces,OU=OU With Spaces",
```

Comment:
Include test strings with more than one RDN. E.g., `Value=CN=Bob, O=Amazon`

Reply:
I did include more than one RDN for the issuer (`OU` as well as `CN`) in this line. I can add another test so that the first key-value pair includes multiple RDNs.